jbpm-human-task/jbpm-human-task-workitems/src/main/java/org/jbpm/services/task/wih/ExternalTaskEventListener.java

```diff
@@ -108,10 +112,12 @@ public void afterTaskCompletedEvent(TaskEvent event) {
         } else {
             logger.error("EE: I've recieved an event but the session is not known by this handler ( "+task.getTaskData().getProcessSessionId()+")");
         }
+        processTaskState(task);
```

Comment:
This is called already on the line 111, do we need to call it again here?

Reply:
This is now corrected.